src/Sql/Insert.php

```diff
@@ -160,7 +160,8 @@ public function getRawState($key = null)
         $rawState = [
             'table' => $this->table,
             'columns' => array_keys($this->columns),
-            'values' => array_values($this->columns)
+            'values' => array_values($this->columns),
```

Comment:
@alextech - I'm slowly understanding more of what you're addressing.  Since I never do anything with Sequence stuff, I'm confident I do not fully grasp the situation fully at all (ha).

If I may offer a different perspective on multiple PRs for this `Insert.php` class: putting multiple Issues' needs into one PR can also be confusing 'down the road.'  If the value of the 'columns' key in `getRawState()` needs to have logic in it that is needed in the Sequence class(es), then it might be better to put such changes into the PR that uses such code changes.

I FULLY understand the dilemma of "PRs from different places depend on one another..."  What I'm not understanding (and this could very well have to do with my not knowing a thing about the Sequence stuff you're talking about) is how adding 'select' to `getRawState()` has any affect on Sequence other than actually making the Select object available?  If anything, it seems that adding 'select' makes this whole Insert object _more_ inspectable while keeping the Select object (and its columns) unique.

Perhaps you can point me to another `zend-db` class where a non-essential object's properties (values, column-names, something else) are 'added' to the essential properties of the containing class?  The other, main class that we use at work is the Select class which does _not_ pull data from classes it 'contains' (ex. Expression objects, TableIdentifier object, etc.) but instead simply returns the object-itself for the user to parse as-desired.

Or to look at it another way, if we change the content of `getRawState('columns')` in this PR, then would that not be a breaking change whereas adding the ability to inspect the Select object - even though this introduces more conditionals in situations like the Sequence stuff you mention - at least gives access to what is needed without breaking how the current method/contract works?

Again, if I am fully misunderstanding the situation you are describing, please just let me know and perhaps we can figure out a solution that accomplishes what you're seeing but - due to my lack of experience - doesn't require me to learn everything about the Sequence problems (grin).